lib/docs/filters/enzyme/clean_html.rb

```diff
@@ -0,0 +1,34 @@
+module Docs
+  class Enzyme
+    class CleanHtmlFilter < Filter
+      def call
+        @doc = at_css('.page-inner > section')
+
+        # Clean headers
+        css('h1').each do |node|
+          node.content = node.content
+        end
+
+        # Make headers on reference pages bigger
```

Comment:
Can you also increase h3s to h2s in the API docs? http://localhost:9292/docs/api/selector